scripts/provision-image-post-update-version.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
+
+import os
+import subprocess
+import sys
+
+INSTALLER_VERSION = "latest"
+
+def enable_installer_service(path):
+    cmd = "systemctl enable clr-provision.service --root={}".format(path)
+    subprocess.run(cmd.split())
+
+
+if __name__ == '__main__':
+    if len(sys.argv) != 2:
+        sys.exit(-1)
+
+    try:
+        enable_installer_service(sys.argv[1])
+    except Exception as exep:
+        print(exep)
+        sys.exit(-1)
+    sys.exit(0)
```

Comment:
Please, no new python scripts.

Remove this file and I'll comment on the replacement command in the post-install of the YAML.